.ci/Jenkinsfile_coverage

```diff
@@ -12,9 +12,10 @@ kibanaPipeline(timeoutMinutes: 240) {
     ]) {
       workers.base(name: 'coverage-worker', size: 'l', ramDisk: false, bootstrapped: false) {
         catchError {
-          kibanaCoverage.runTests()
-          kibanaTeamAssign.load('team_assignment', "### Upload Team Assignment JSON")
-          handleIngestion(TIME_STAMP)
+//          kibanaCoverage.runTests()
+//          kibanaTeamAssign.load('team_assignment', "### Upload Team Assignment JSON")
+//          handleIngestion(TIME_STAMP)
+          kibanaTeamAssign.generateCodeOwners('.github/CODEOWNERS', "### Generate CODEOWNERS")
```

Comment:
This needs to be removed

Reply:
Yeah I was wondering how we wanna handle this

> The CODEOWNERS file isn't updated, and the validation of the codeowners file isn't happening as we discussed

I thought the validation would live in the same pr as the ci check.  No?

Reply:
Where do we wanna handle this guy?  kibanaPipeline.groovy?